### DISCUSSION TITLE
Filtering baaed on SharePoint Online column of type DateOnly will not return any data

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#3922] [Q&A]
**I have a SharePoint date field of type DateOnly as follow with internal name = READYDATE:-**

![image](https://github.com/user-attachments/assets/9bbdb18e-b521-4151-a28e-50865750f011)

**And then i added a RefinableDate00 and  connect it to the related crawl property (Ows_READYDATE), as follow: -**

![image](https://github.com/user-attachments/assets/a55a0783-7bf0-4ec4-9830-089a6283ecd9)

**I did a re-index for the site 30 hours ago.** 

**Then i added a Search Result web part, as follow:-**
![image](https://github.com/user-attachments/assets/ff486623-3c6f-490c-b104-9e1949da9926)

**Then i added a slot representing the refinabledate00, as follow:-**

![image](https://github.com/user-attachments/assets/bc4bd53d-b6c1-4767-a376-207b584e4f8c)

**and i am showing the field inside the Details List using this expression:-**

{{getDate (slot item @root.slots.ReadyDate) 'LL'}}

**and the column values are showing well as follow:-**

![image](https://github.com/user-attachments/assets/7afa7b67-4fdc-4c01-897b-9610d4edcb1a)

**then i added a Filter web part and i configure a single column which is the RefinableDate00, as follow:-**

![image](https://github.com/user-attachments/assets/4d84a118-60d9-41e9-a041-7e46c65447e6)

**but when i select any date range i will always get no results, as follow:-**

![image](https://github.com/user-attachments/assets/84d44797-a88d-4c7d-8a56-04083c7df4b6)

**so what could be the reason, of course i connect the Search result web part with the Filter web part..**

**Comment by kasperbolarsen** [comment]
Have you verified the data using the Graph Explorer?

**Reply by mvcsharepointdev** [reply]
@kasperbolarsen no , but i am not sure why this is not working at my end? as seems i have everything setup correctly
